de1plus/misc.tcl

```diff
@@ -140,6 +140,9 @@ proc make_de1_dir {srcdir destdirs} {
         de1_de1.tcl *
         device_scale.tcl *
         event.tcl *
+	event.tcl *
+
+	build-info *
```

Comment:
Please create a placeholder file so that a clean clone is always a full app

Reply:
Confirmed that logic checks for the file being readable before trying to open it. As a result, this will function if `build-info` is not present on the user's machine.

Reply:
the updater will not create the distro and the download will fail if not present on the server

Reply:
Sounds like a build system problem to me. `build-info` gets created, along with the proper `version.tcl` file. Both are in the list of files to be included. 

What kind of build system can't handle dynamically generated artifacts?